### PR TITLE
Feature/get user work edu

### PR DESCRIPTION
**Pull request by pavlo-chernykh**
what was done:
fixes to match fe
added queries to delete data from work and education before adding new (to prevent adding new info without deleting old one first)
added get education endpoint to get education and send it to FE, here's swagger:
image
added get workhistory endpoint to get workhistory and send it to FE, here's swagger:
image

### CODE REVIEW

**Review by SergKukh — APPROVED** [review]

**Review by KashyretsIvanna — APPROVED** [review]

File: src/modules/user/user.service.ts [hunk 1]
```diff
@@ -284,6 +296,38 @@ export class UserService {
     }
   }
 
+  async getWorkInfo(user: UserDto): Promise<WorkHistory[]> {
+    try {
+      const Workhistories = await this.workHistoryRepository
```
Comment (lyudad) [inline src/modules/user/user.service.ts]: no capital letters for the constants, where have you found it? this week you are 3d or 4th who follow this approach. if you go with upper case - everywhere uppercase (but senior developers will hate you:) ) if go with lover case - only lowercase

File: src/modules/user/user.service.ts [hunk 2]
```diff
@@ -284,6 +296,38 @@ export class UserService {
...
+      if (error instanceof HttpException) {
+        throw error;
+      }
+      throw new InternalServerErrorException();
+    }
+  }
+
+  async getEducationInfo(user: UserDto): Promise<Education[]> {
+    try {
+      const Educations = await this.educationRepository
```
Comment (lyudad) [inline src/modules/user/user.service.ts]: camon

File: src/modules/user/user.service.ts (hunk 1 above)
Comment (pavlo-chernykh) [reply src/modules/user/user.service.ts]: yeah...that's awful, agree! Fixed

File: src/modules/user/user.service.ts (hunk 2 above)
Comment (pavlo-chernykh) [reply src/modules/user/user.service.ts]: fixed

**Review by lyudad — APPROVED** [review]

**Pull request merged**